cmd/end2endtest/helpers.go

```diff
@@ -31,6 +31,14 @@ type voteInfo struct {
 	keys         []vapi.Key
 }
 
+var processStatusName = map[int32]string{
```

Comment:
This is wrong, since 0 is `UNKNOWN`.

Furthermore, instead of declaring this map here, you might use the map from `models.ProcessStatus_name`.

Models is an import from "go.vocdoni.io/proto/build/go/models"